static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -10,64 +10,144 @@ const config = (config_ as unknown) as {
 	get: jest.MockedFunction<(s: string, d: boolean) => boolean>;
 };
 
-jest.mock('lib/raven', () => ({
-	captureMessage: jest.fn(),
-}));
+MockDate.set(new Date('2021-01-01T00:00:00Z'));
+
+const CODE_ENDPOINT = '//logs.code.dev-guardianapis.com/log';
+const PROD_ENDPOINT = '//logs.guardianapis.com/log';
+const TEST_URL = 'http://testurl.theguardian.com/';
+
+const owner = 'commercial.sentinel';
+const mockDate = new Date();
+const defaultEvent: SentinelLoggingEvent = {
+	received_timestamp: mockDate,
+	received_date: mockDate.toISOString().slice(0, 10),
+	label: owner,
+};
 
 jest.mock('../../lib/config', () => ({
 	get: jest.fn(),
 }));
 
+beforeEach(() => {
+	navigator.sendBeacon = jest.fn();
+});
+
 afterEach(() => {
 	jest.clearAllMocks();
 });
 
+afterAll(() => {
+	MockDate.reset();
+});
+
 describe('sentinel', () => {
-	test('does not send a message when switches.sentinelLogger is false', () => {
+	test('should not send an event when switches.sentinelLogger is false', () => {
 		config.get.mockReturnValue(false);
-		amIUsed('moduleName', 'functioName');
-		expect(raven.captureMessage).not.toHaveBeenCalled();
+		amIUsed('moduleName', 'functionName');
+		expect(navigator.sendBeacon as jest.Mock).not.toHaveBeenCalled();
```

Comment:
Hopefully setting the most at the top-level might prevent the need for the type assertion.

```suggestion
		expect(navigator.sendBeacon).not.toHaveBeenCalled();
```

Reply:
It works, thanks! But it still requires type assertion when `.mock.calls` is appended. 
I'll remove the type assertion from the other tests.